sphinx/texinputs/sphinx.sty

```diff
@@ -539,12 +539,56 @@
 }
 
 % Notices / Admonitions
-%
 
+% Some are quite plain
+\newenvironment{sphinxlightbox}{%
+  \par\allowbreak
+  \noindent{\color{spx@notice@bordercolor}%
+            \rule{\linewidth}{\spx@notice@border}}\par\nobreak
+  {\parskip\z@skip\noindent}%
+  }
+  {%
+  \par
+  % counteract previous possible negative skip (French lists!):
+  % (we can't cancel that any earlier \vskip introduced a potential pagebreak)
+  \ifdim\lastskip<\z@\vskip-\lastskip\fi
+  \nobreak\vbox{\noindent\kern\@totalleftmargin
+      {\color{spx@notice@bordercolor}%
+       \rule[\dimexpr.4\baselineskip-\spx@notice@border\relax]
```

Comment:
@tk0miya for perfect same rendering as earlier code I should use here 

    \rule[\dimexpr.4\baselineskip+0.5pt-\spx@notice@border\relax]

because default thickness is `0.5pt` and earlier code used `.4\baselineskip` to shift upwards the rule respective to baseline.